spec.md

```diff
@@ -864,6 +867,32 @@ message GetCapacityResponse {
 }
 ```
 
+#### `ProbeController`
+
+A Controller Plugin MUST implement this RPC call.
+The Plugin SHALL assume that this RPC will be executed on the controller where the volume will be created.
```

Comment:
There's a bunch of wording here that doesn't make sense for a controller service since it could be either a central service, or a distributed one. Want to take a stab at hacking the wording a bit? If not, I'll happily make a suggestion.